frontend-react/src/hooks/UseValueSets.ts

```diff
@@ -92,10 +92,15 @@ export const useValueSetsMeta = (
     dataTableName: string = LookupTables.VALUE_SET
 ): ValueSetsMetaResponse => {
     const { authorizedFetch, rsUseQuery } = useAuthorizedFetch<LookupTable[]>();
+    const memoizedDataFetch = useCallback(
```

Comment:
Why does this need to be memoized?  Even if the `authorizedFetch` reference changes, react-query should still prevent another request if the query key doesn't change, no?

Reply:
This was from earlier on when troubleshooting errors (value sets was a real trouble area) and I was throwing everything at the wall to see what would stick (aka stop erroring). I can revert if needed.